pkg/epinio/models/epinio-resource.js

```diff
@@ -57,4 +61,38 @@ export default class EpinioResource extends Resource {
       throw epinioExceptionToErrorsArray(e);
     }
   }
+
+  // Close APPS logs on namespace deletions
+  async _closeLogsOnDelete(namespace) {
+    try {
+      const namespaces = await this.$dispatch('findAll', { type: this.type, opt: { force: true } });
+
+      // Find new namespace
+      const current = namespaces.filter(n => n.id === namespace)[0];
```

Comment:
once moved into models this won't be needed, but for future reference to get a resource you can use just `find` to get a single resource. Both find actions will fetch the resource if missing. In this case though we know we have the namespace so we can just use the syncronouse getter (for example for kube namespaces it would be something like `this.$store.getters['cluster/byId'](NAMESPACE, name) `)